ivy/functional/frontends/numpy/statistics/order_statistics.py

```diff
@@ -35,6 +36,13 @@ def _cpercentile(N, percent, key=lambda x: x):
     return d0 + d1
 
 
+@to_ivy_arrays_and_back
+def ptp(a, axis=None, out=None, keepdims=False):
+    x = ivy.max(a, axis=axis, keepdims=keepdims)
+    y = ivy.min(a, axis=axis, keepdims=keepdims)
+    return ivy.subtract(x, y)

```

Comment:
Also the returned dtypes should be the same as input ones, so you'll need to ensure that at the return🙂

Reply:
i'm a little confused on this. the numpy docs don't have `dtype` as a param so not sure where that is coming from.